[DOCS-7579] NDM profile onboarding (#23435)
DataDog/documentation: content/en/network_monitoring/devices/guide/device_profiles.md

3. Provide your device profile a name, vendor information(optional) and description (optional).
4. Select the `SysObjectID`. This is what is used to match network devices to the device profiles that define what is collected and monitored from each device. 

{{< img src="/network_device_monitoring/profile_onboarding/Sys_object_ID_Field.png" alt="The Network Device profile creation page showing the Sys Object ID Dropdown" style="width:100%;">}}

### Global Tags

Add global tags for more advanced and granular options, which allows you to assign a weight to a specific metric.

{{< img src="/network_device_monitoring/profile_onboarding/Add_global_tags.png" alt="The Network Device profile creation page showing the Add Global Tags dropdown" style="width:100%;">}}

#### Advanced Options

| Modification       | Description   |
| ------------- | ------------- |
| No Modification | The device's returned value will be used directly as the tag value. |
| Format | This can be [mac_address][5] or [ip_address][6]. |
| Extract Value | A regular expression used to [extract][7] the tag value from the SNMP value provided by the device. |

> | Mapping | This is described [here][8]. Note: our current docs may imply this can ONLY be used for table tags, which needs correction. |

Reviewer: How long will it take to correct this? Since users won't know this is separate docs, sounds like we're discrediting ourselves. Recommend removing and just fixing the incorrect statement.

```suggestion
| Mapping | This is described [here][8]. |
```

### Scalar metrics

See [Metrics Advanced Options](#metrics-advanced-options).

### Tabular metrics

See [Metrics Advanced Options](#metrics-advanced-options).

### Add Tabular Tags

Adding tags to tabular metrics is similar to adding global tags, with two additional options:

1. Select whether the tag value originates from an `OID` value or a segment of the table index. If `Index` is chosen as the source, an index position must be specified, which then becomes the tag.

> <details>

Reviewer: I really like how this looks! I wonder if this can also indent so it's in line with number 1.

  If you want to report the current on a PDU branch, you could add `cpiPduBranchCurrent` (`OID 1.3.6.1.4.1.30932.1.10.1.3.110.1.3`, from the branch table) as a tabular metric. To tag this metric with the PDU name, add `cpiPduName` as a tag (`OID 1.3.6.1.4.1.30932.1.10.1.2.10.1.3`, from the main table), then add an index transform with `start:1` and `end:7`. This means the branch current metric from the branch table indexed with `1.6.0.36.155.53.3.246` would be tagged using the name from the main table indexed with `6.0.36.155.53.3.246`. 
  
  For more information see [Using a column from a different table with different indexes][10].

</details>

### Metrics Advanced Options

The advanced options for scalar and tabular metrics are the same:

[Metric Type][11]
: One of `gauge`, `rate`, `monotonic_count`, or `monotonic_count_and_rate`. 

[Scale Factor][12]
: Before being transmitted to Datadog, the extracted value is multiplied by this factor.

[Extract Value][7]
: This is the same as the [advanced options](#advanced-options) for global tags.

### Apply a profile to created devices

> 1. Once you've saved a profile, navigate back to the [profile home page][4] and select the **Download All Profiles** option. This allows you to download the `yaml` files for the profiles you've created. 

Reviewer: ```suggestion
1. After you save a profile, navigate back to the [profile home page][4] and select the **Download All Profiles** option. This allows you to download the `yaml` files for the profiles you created. 
```

> 2. Place the `yaml` files in the [profile directory][13] on each of the relevant installed Agent's.

Reviewer: ```suggestion
2. Place the `yaml` files in the [profile directory][13] on each of the relevant installed Agents.
```

3. Restart the Datadog Agent.

> 4. To ensure the profiles you've created are accurate, confirm that NDM is receiving metrics from the matched devices as expected.

Reviewer: ```suggestion
4. To ensure the profiles you created are accurate, confirm that NDM is receiving metrics from the matched devices as expected.
```

> **Need updated screen shot** - This is a placeholder

Reviewer: Flagging placeholder
